src/chrome/content/rules/Asciinema.org.xml

```diff
@@ -1,27 +1,17 @@
 
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://www.asciinema.org/ => https://www.asciinema.org/: (51, "SSL: no alternative certificate subject name matches target host name 'www.asciinema.org'")
-
-	CDN buckets:
-
-		- asciinema-blog.herokuapp.com
-
-			- blog
-
-
-	Problematic subdomains:
-
-		- blog *
-
-	* Works; mismatched, CN: *.herokuapp.com
+	Invalid certificate:
+		www.asciinema.org
+		blog.asciinema.org
 
 -->
-<ruleset name="Asciinema.org (partial)" default_off='failed ruleset test'>
+<ruleset name="Asciinema.org (partial)">
 
 	<target host="asciinema.org" />
-	<target host="blog.asciinema.org" />
 	<target host="www.asciinema.org" />
+	<target host="blog.asciinema.org" />
+	<target host="staging.asciinema.org" />
+
 		<!--
 			Avoid user-visible paths:
```

Comment:
Why are we doing this?

Reply:
I'm guessing it's to avoid sending the user to a "strange" URL. They may freak out seeing they landed on a different domain than the one they requested.

Reply:
I fine with removing the complicated exclusion in favor of https and simplicity.